charts/k8ssandra-cluster/templates/cassdc.yaml

```diff
@@ -12,6 +12,7 @@ spec:
   clusterName: {{ .Values.k8ssandra.clusterName }}
   serverType: cassandra
   serverVersion: {{ .Values.k8ssandra.cassandraVersion | quote }}
+  dockerImageRunsAsCassandra: false
```

Comment:
This should work around the logic in cass-operator [here](https://github.com/datastax/cass-operator/blob/master/operator/pkg/images/images.go#L168-L179):
Do we need to open an issue to address it in cass-operator as well?

Reply:
I created https://github.com/datastax/cass-operator/issues/353.